src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java

```diff
@@ -412,28 +403,28 @@ public void writeCsvResponse(Result[] resultSet, HttpServletResponse servletResp
     switch (elemGeom) {
       case BBOX:
         if (deletionHandling) {
-          return new org.wololo.geojson.Feature(emptyPolygon, properties);
+          return new org.wololo.geojson.Feature(null, properties);
         }
         Envelope envelope = geometry.getEnvelopeInternal();
         OSHDBBoundingBox bbox = OSHDBGeometryBuilder.boundingBoxOf(envelope);
         outputGeometry = OSHDBGeometryBuilder.getGeometry(bbox);
         break;
       case CENTROID:
         if (deletionHandling) {
-          return new org.wololo.geojson.Feature(emptyPoint, properties);
+          return new org.wololo.geojson.Feature(null, properties);
         }
         outputGeometry = geometry.getCentroid();
         break;
       case RAW:
       default:
         if (deletionHandling && geometry.getGeometryType().contains("Polygon")) {
-          return new org.wololo.geojson.Feature(emptyPolygon, properties);
+          return new org.wololo.geojson.Feature(null, properties);
         }
         if (deletionHandling && geometry.getGeometryType().contains("LineString")) {
-          return new org.wololo.geojson.Feature(emptyLine, properties);
+          return new org.wololo.geojson.Feature(null, properties);
         }
         if (deletionHandling && geometry.getGeometryType().contains("Point")) {
-          return new org.wololo.geojson.Feature(emptyPoint, properties);
+          return new org.wololo.geojson.Feature(null, properties);
```

Comment:
I don't see the point of the repeated `if` cluster here to always return the same result. Also, `GeometryCollections` fall through here

Reply:
PS: the `if (deletionHandling) { … }` could even be performed before the `switch (elemGeom)` statement, right?

Reply:
Yes, you're right. Added that now as well.